frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/result-tab/result-tab-row.scss

```diff
@@ -0,0 +1,17 @@
+.kubevirt-create-vm-modal___result-tab-row-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.kubevirt-create-vm-modal___result-tab-row-container--error > button {
+    color: #a30000 !important;
+}
+
+.kubevirt-create-vm-modal__result-tab-row {
+  text-align: left;
+}
+
+.kubevirt-create-vm-modal__result-tab-row--error {
+  background-color: #ffe6e6;
```

Comment:
How did we get these colours? Aren't they defined by patternfly?

Reply:
they were taken from pf3 notification error

Reply:
you can use `--pf-global--danger-color--200` here instead

Reply:
fixed the dark red, let's proceed with this one and patch it later once the light red is available